Tests/Form/Type/AssertFormReferenceCompatibleWithSymfony28Trait.php

```diff
@@ -0,0 +1,37 @@
+<?php
+
+/*
+ * This file is part of the Sonata Project package.
+ *
+ * (c) Thomas Rabaix <thomas.rabaix@sonata-project.org>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Sonata\MediaBundle\Tests\Form\Type;
+
+trait AssertFormReferenceCompatibleWithSymfony28Trait
```

Comment:
@Soullivaneuh 
Ok, I'm not sure to understand.
Do you ask to me to just remove the trait and use it in Abstracts ? Or remove all Abstracts too ? 
Or are you telling me all tests was wrong ?

I understand the final goal is to test if the FormType works. But it's clearly out of the scope of this PR to test it. Maybe I could do a separated PR to do this, but this will not be done now.